lib/screens/HomeScreen.tsx

```diff
@@ -517,7 +539,33 @@ const HomeScreen: React.FC<HomeScreenProps> = ({ navigation, route }) => {
     );
   };
 
+  //Part of searchbar:
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const handleSearch = (query: string) => {
+    setSearchQuery(query);
+  };
+  const formatDate = (date: Date) => {
+    const day = date.getDate().toString();
+    const month = (date.getMonth() + 1).toString(); // Months are zero-based
+    const year = date.getFullYear();
+    return `${month}/${day}/${year}`;
+  };
+
+  const filteredNotes = useMemo(() => {
+    return notes.filter(note => {
+      const noteTime = new Date(note.time);
+      const formattedTime = formatDate(noteTime);
+      
+      return note.title.toLowerCase().includes(searchQuery.toLowerCase()) || 
+             formattedTime.includes(searchQuery.toLowerCase());
+    });
+  }, [notes, searchQuery]);
+
+
+
   return (
+ 
```

Comment:
I see that the idea behind searching for entries in the search bar is better now, as you looking for more details and information when text is inputting into the search bar.